### PR TITLE
Made some code better

### PR DESCRIPTION
**Pull request by bicsi**

### CODE REVIEW

**Comment by ericpts** [comment]
Please use more descriptive sentences than 
`Made some code better`

**Comment by ericpts** [comment]
Also resolve the conflicts

File: src/bots/Bicsi.bot [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 #include "bot.h"
 #include <algorithm>
+#include <map>
```
Comment (ericpts) [inline src/bots/Bicsi.bot:6]: what is this used for?

File: src/util/debug.h [hunk 2]
```diff
@@ -3,6 +3,6 @@
 #include <stdio.h>
 
 #define DEBUG 1
-#define debug(...) if(DEBUG) {fprintf(stderr, __VA_ARGS__);}
+#define debug(...) if(DEBUG) fprintf(stderr, __VA_ARGS__); else
```
Comment (ericpts) [inline src/util/debug.h:6]: why this change?

File: src/bots/Bicsi.bot (hunk 1 above)
Comment (bicsi) [inline src/bots/Bicsi.bot:6]: Shhhh
